pkgs/applications/editors/neovim/qt.nix

```diff
@@ -4,13 +4,13 @@
 let
   unwrapped = stdenv.mkDerivation rec {
```

Comment:
shouldn't this follow the recommendation from https://github.com/NixOS/nixpkgs/issues/65399 and use the mkDerivation from qt rather than from stdenv ?

Reply:
Done